livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/CommentsBox.kt

```diff
@@ -0,0 +1,38 @@
+package io.getstream.livestream.compose.ui
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import io.getstream.chat.android.compose.ui.messages.list.MessageList
+import io.getstream.chat.android.compose.viewmodel.messages.MessageComposerViewModel
+import io.getstream.chat.android.compose.viewmodel.messages.MessageListViewModel
+
+/**
+ * A view component to build a custom message box with consists of [MessageList]
+ * and a background scrim
+ */
+@Composable
+fun CommentsBox(
+    modifier: Modifier = Modifier,
+    composerViewModel: MessageComposerViewModel,
+    listViewModel: MessageListViewModel
+) {
+    Box(
```

Comment:
We don't need a box if we're using a column to display all of our items.

Reply:
Tweaked it a bit , we longer need a custom component that wraps our MessageList and compose

ref: `streams` package